src/cosalib/container_manifest.py

```diff
@@ -19,6 +19,22 @@ def create_local_container_manifest(repo, tag, images) -> dict:
                            capture_output=True).stdout
     return json.loads(manifest_info)
 
```

Comment:
Need an extra line here ? CI test complains of 2 blank lines required before every function.